pennylane/qchem/convert.py

```diff
@@ -196,7 +202,7 @@ def _ps_to_coeff_term(ps, wire_order):
     return coeffs, ops_str
 
 
-def _pennylane_to_openfermion(coeffs, ops, wires=None):
+def _pennylane_to_openfermion(coeffs, ops, wires=None, tol=1.0e-16):
```

Comment:
Why did we need to introduce a tolerance?

Reply:
To allow users to remove the imaginary part from the coefficients in a pennylane `LinearCombination` operator, for example if the imaginary part is really small.